app/console.go

```diff
@@ -72,6 +74,13 @@ func RunConsole(a *App) *ishell.Shell {
 			editUser(ctx, a)
 		},
 	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "cryptowallet",
```

Comment:
Fingerbreakingly painful command 😂, how about encrypt?

Reply:
Havnt has a chance to look at the whole PR but I prefer cryptowallet to encrypt. It might be slightly longer but it’s more accurate. Plus the CLI has autocomplete.

Reply:
Yeah encrypt is a little vague to the user. What am I encrypting? But I agree it would be better shorter if anyone can think of anything

Reply:
wallet encrypt? I don’t think cryptowallet is very clear. Maybe I’m splitting hairs.

Reply:
`secure`, `secure-wallet`, or `safe-wallet` maybe?